src/Nethermind/Nethermind.Specs.Test/OverridableReleaseSpec.cs

```diff
@@ -150,5 +151,6 @@ public long Eip1559TransitionBlock
         public bool IsEip1153Enabled => _spec.IsEip1153Enabled;
         public bool IsEip3675Enabled => _spec.IsEip3675Enabled;
         public bool IsEip3651Enabled => _spec.IsEip3651Enabled;
+        public bool IsEip3860Enabled { get; set;}
```

Comment:
I think this should be `public bool IsEip3860Enabled => _spec.IsEip3860Enabled `

Reply:
I defined it this way, so I can override the value in tests - to cover EIP 3860 on/off. If there's a better way, I'm happy to change.

Reply:
Great - thanks. Changes pushed.